Sources/Networking/Responses/CustomerInfoResponse.swift

```diff
@@ -74,16 +78,34 @@ extension CustomerInfoResponse {
         var productIdentifier: String
         var purchaseDate: Date?
 
+        @IgnoreEncodable @IgnoreHashable
+        var rawData: [String: Any]
+
     }
 
 }
 
 // MARK: -
 
 extension CustomerInfoResponse.Subscriber: Codable, Hashable {}
-extension CustomerInfoResponse.Entitlement: Codable, Hashable {}
 extension CustomerInfoResponse.Subscription: Codable, Hashable {}
 
+extension CustomerInfoResponse.Entitlement: Hashable {}
+extension CustomerInfoResponse.Entitlement: Encodable {}
+extension CustomerInfoResponse.Entitlement: Decodable {
+
+    init(from decoder: Decoder) throws {
+        let container = try decoder.container(keyedBy: CodingKeys.self)
+
+        self.expiresDate = try container.decodeIfPresent(Date.self, forKey: .expiresDate)
+        self.productIdentifier = try container.decode(String.self, forKey: .productIdentifier)
+        self.purchaseDate = try container.decodeIfPresent(Date.self, forKey: .purchaseDate)
+
+        self.rawData = decoder.decodeRawData()
```

Comment:
This is the only downside, but it's not too bad. We must manually implement this method. But reusing `decodeRawData()` it's pretty clean.